docker_plugin/tasks.py

```diff
@@ -42,20 +42,19 @@ def create_container(params, daemon_client=None, **_):
         as provided to the start function. The create function
         will pass only the dict keys as the ports parameter and
         the start function will pass the pairs as port bindings.
-    :param daemon_client: optional configuration for client creation
+    :param connection_credentials: optional configuration for client creation
     """
 
-    daemon_client = daemon_client or {}
-    client = docker_client.get_client(daemon_client)
+    client = docker_client.get_client(connection_credentials)
 
     if ctx.node.properties['use_external_resource']:
         if 'name' not in ctx.node.properties:
             raise NonRecoverableError(
                 'Use external resource, but '
                 'no resource id provided.')
-        ctx.instance.runtime_properties['container_id'] = \
+        ctx.instance.runtime_properties['container_id'] = (
```

Comment:
There's no convention, at all. There are only tools restrictions (like testtools, flake8). Also, please refer to Cloudify development docs where the restriction is being mentioned. Until that moment, it would remain the same. Thanks.